projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration.ts

```diff
@@ -205,10 +209,22 @@ export function isImageNotSelected(
   cy.get(`#${valueId}`).should('not.be.checked');
 }
 
+export function goToConfigPage(configuratorType, product) {
+  cy.visit(
+    `/electronics-spa/en/USD/configure${configuratorType}/product/entityKey/${product}`
+  ).then(() => {
+    isConfigPageDisplayed();
+  });
+}
+
 export function isGroupMenuDisplayed() {
   cy.get('cx-config-group-menu').should('be.visible');
 }
 
```

Comment:
Plese, add a comment. Do you mean a product header or?
Please, name the method accordingly.